tests/test_functional.py

```diff
@@ -610,3 +610,68 @@ def test_empty_hyperslabs(pcic_data_portal, projection, length):
 
     nc.close()
     os.remove(f.name)
+
+    
+@pytest.mark.bulk_data
+@pytest.mark.parametrize(("file,expected_mean"),
+                         [('pr+tasmax+tasmin_day_BCSD+ANUSPLIN300+MRI-CGCM3_historical+rcp85'
+                           '_r1i1p1_19500101-21001231.nc.nc?tasmax[55000:55151][239:239][210:210]&', 
+                           8.052631378),
+                           ('/pr+tasmax+tasmin_day_BCSD+ANUSPLIN300+HadGEM2-ES_historical+rcp85'
+                            '_r1i1p1_19500101-20991230.nc.nc?tasmax[13467:13467][150:178][877:900]&',
+                            -6.884425163)
+                         ])
+def test_record_variables(pcic_data_portal, file, expected_mean):
```

Comment:
As part of addressing the PEP8 nonconformities, let me suggest the following layout of `parametrize`:

```
@pytest.mark.parametrize('file,expected_mean', [
    ('pr+tasmax+tasmin_day_BCSD+ANUSPLIN300+MRI-CGCM3_historical+rcp85'
     '_r1i1p1_19500101-21001231.nc.nc?tasmax[55000:55151][239:239][210:210]&', 
     8.052631378),
     ('/pr+tasmax+tasmin_day_BCSD+ANUSPLIN300+HadGEM2-ES_historical+rcp85'
     '_r1i1p1_19500101-20991230.nc.nc?tasmax[13467:13467][150:178][877:900]&',
     -6.884425163)
])
```

Note: Parens around the string first argument are redundant. `parameterize` splits a comma-separated string into a tuple for you, and I suspect this is a blind copy-paste from some code where the developer did the splitting manually (e.g., `('arg1', 'arg2')` vs. `'arg1, arg2'`).